org.freedesktop.LinuxAudio.Plugins.Calf.json

```
@@ -0,0 +1,50 @@
+{
+    "id": "org.freedesktop.LinuxAudio.Plugins.Calf",
+    "branch": "21.08",
+    "runtime": "org.freedesktop.LinuxAudio.BaseExtension",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk//21.08",
+    "build-extension": true,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/app/extensions/Plugins/Calf",
+        "prepend-ld-library-path": "/app/extensions/Plugins/Calf/lib",
+        "prepend-pkg-config-path": "/app/extensions/Plugins/Calf/lib/pkgconfig"
+    },
+    "cleanup": [
+        "/bin",
+        "/share/applications",
+        "/share/bash-completion",
+        "/share/doc",
+        "/share/icons",
+        "/share/man"
+    ],
+    "modules": [
+        "shared-modules/linux-audio/fluidsynth2-static.json",
+        "shared-modules/linux-audio/lv2.json",
+        {
+            "name": "calf",
+            "config-opts": [
+                "--with-lv2",
+                "--with-lv2-dir=/app/extensions/Plugins/Calf/lv2"
+            ],
```

```suggestion
            ],
            "build-options": {
                "arch": {
                    "x86_64": {
                        "config-opts": [
                            "--enable-sse"
                        ]
                    }
                }
            },
```

This how it should be done to enable-sse on x86_64 (the fact they don't do it automatically boggle my mind, there is no x86_64 CPU that don't have SSE)